### PR TITLE
Use new PostgreSQL backend by default

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Let's go!

Documentation is not updated yet because https://docs.ferretdb.io is not versioned, and we don't want to show v1.13 docs right after the v1.12 release.

## Readiness checklist

- [x] I added/updated unit tests (and they pass).
- [x] I added/updated integration/compatibility tests (and they pass).
- [x] I added/updated comments and checked rendering.
- [x] I made spot refactorings.
- [ ] I updated user documentation.
- [x] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

**Review by chilagrow — DISMISSED** [review]
:rocket:

**Review by rumyantseva — DISMISSED** [review]
:shipit:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -132,10 +132,10 @@ The `internal` subpackages contain most of the FerretDB code:
 - `handlers/sjson` provides converters from/to SJSON for built-in and `types` types.
   SJSON adds some extensions to JSON for keeping object keys in order,
   preserving BSON type information in the values themselves, etc.
-  It is used by `sqlite` and `pg` handlers.
+  It is used by `sqlite` and old `pg` handlers.
```
Comment (chilagrow) [inline CONTRIBUTING.md:135]: We use it in new `postgresql` backend too, for mashal and stuff. Is that a problem?

**Review by chilagrow — APPROVED** [review]

**Review by chilagrow — APPROVED** [review]

**Review by noisersup — APPROVED** [review]
The previous review was about docs being not relevant, but I've checked the PR description Looks good to me! (Also sorry @chilagrow I re-requested review from you by accident)

**Review by chilagrow — APPROVED** [review]

**Review by AlekSi — COMMENTED** [review]
hm…

**Comment by AlekSi** [comment]
That PR broke GitHub…

**Pull request closed without merging**